ml-proto/src/spec/memory.ml

```diff
@@ -91,7 +91,8 @@ let load64 mem a =
   Int64.logor (Int64.of_int32 (load32 mem a)) (Int64.shift_left (Int64.of_int32 (load32 mem (a+4))) 32)
 
 let store8 mem a bits =
-  !mem.{a} <- Int32.to_int (Int32.logand bits (Int32.of_int 255))
+  (* Store "bits" at byte index a; this implicitly truncates all but the lowest 8 bits of "bits". *)
```

Comment:
I apologize for being pedantic, but "truncate" in wasm spec lingo means discarding least-significant digits of a value. How about "this stores just the low 8 bits of "bits", discarding the rest"?